test/unit/components/queryBuilder/queryOverview/queryOverviewControl/diseaseListing/diseaseListing_spec.js

```diff
@@ -65,7 +65,8 @@ describe("UNIT::component: diseaseListing:", () => {
     let sampleCount_Val = findIn(element, ".js-test-sampleCount").text();
 
     expect(sampleCountEl).toBeDefined();
-    var sampleCount = parentScope.paramList[0].samples.length;
+    console.log(sampleCount_Val);
```

Comment:
This looks like a debug line. Did you mean to leave it?

Reply:
oops! 🙈